openjdk/openjdk.mk

```diff
@@ -28,7 +28,11 @@ ifeq ($(CYGWIN),1)
 endif
 EXTRA_JTREG_OPTIONS += -concurrency:$(NPROCS)
 
-JTREG_BASIC_OPTIONS += -agentvm
+ifeq ($(OS),OS/390)
+    JTREG_BASIC_OPTIONS += -othervm
```

Comment:
Could also add a comment if this is a temporary case due to -agentvm option not working on zos (that way there may be a chance we see it and change it later when it is working (curiosity, is there an issue captured somewhere describing what blocks it from working?)

Reply:
we have no idea what the issue is yet, whether fundamental or a bug as there is no zOS agent diagnostics... i'm hoping a bug and once we fix all the sanity failing tests then maybe one of them fixes this....